frontend/src/modules/report/pages/report-view-page-public.vue

```diff
@@ -50,14 +70,22 @@
       >
         <div class="w-full mt-8">
           <app-report-member-template
+            v-if="
+              currentTemplate.name === MEMBERS_REPORT.name
+            "
             :is-public-view="true"
+            :filters="{
+              platform,
+              teamMembers
+            }"
           />
         </div>
       </app-page-wrapper>
       <!-- Custom Report -->
       <div v-else class="max-w-5xl flex flex-grow mx-auto">
         <app-report-grid-layout
           v-model="report"
+          :is-public-view="true"
```

Comment:
Was this hardcoded on purpose?

Reply:
Yes, maybe the name itself is not the best but basically this boolean indicates that the report is being seen in the public view. Since this is the `report-view-page-public.vue` component this is always the public view

Should I update the prop name to something else?

Reply:
Makes sense!

>Should I update the prop name to something else?
No need, just wanted to double-check if it intended 👍